test/integration/guest_user_test.rb

```diff
@@ -85,7 +85,44 @@ def setup
     end
 
     within('#price') do
-      assert page.has_content?(1200)
+      assert page.has_content?('$12.00')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.